File: week_6_Python/lecture/mario.py
# from cs50 import get_int

def main():
    height = get_height()
    for i in range(height):
        print("#")

    for i in range(4):
        print("?", end="")
    print()


def get_height():
    while True:
        try:
            n = int(input("Height: "))
            if n > 0:
                return n
        except ValueError:
            print("Not an integer")


main()

File: week_1_C/pset_1/mario-more/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //ask for height
    int height;
    do
    {
        height = get_int("height: ");
    }
    while (height < 1 || height > 8);

    //printing #s
    for (int i = 0; i < height; i++)
    {
        for (int k = 1; k < height - i; k++)
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("  ");
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}